jerry-core/debugger/debugger.c

```diff
@@ -522,6 +523,22 @@ jerry_debugger_breakpoint_hit (uint8_t message_type) /**< message type */
 } /* jerry_debugger_breakpoint_hit */
 
 /**
+* Suspend execution for received time
+*/
+void jerry_debugger_sleep (int milliseconds) /**< Suspending time */
```

Comment:
Moreover, I don't think that this function needs to be exported from this module. It can be kept static, as it's only accessed from this file.